src/styles/scss/base/_variables.scss

```diff
@@ -1,4 +1,17 @@
-$font-main: 'Open Sans', sans-serif;
+// Palette
+$color-main: #eb6e64;
+$color-white: #fff;
 
-$color-main: #EB6E64;
-$color-white: #FFF;
+// Typography
+// Font-families
+$font-main: 'Lato', 'Open Sans', sans-serif;
+
+// Font-sizes
+$fz-xxs: 0.75rem; // 12px
+$fz-xs: 0.875rem; // 14px
+$fz-s: 1rem; // 16px
+$fz-m: 1.125rem; // 18px
+$fz-l: 1.5rem; // 24px
+$fz-xl: 2rem; // 32px
+$fz-xxl: 4.25rem; // 68px
+$fz-giant: 4.25rem; // 84px
```

Comment:
```scss
$fz-xxl: 4.25rem; // 68px
$fz-giant: 4.25rem; // 84px
```
Looks like both `xxl` & `giant` font-size has same with `4.25rem` here. `$fz-giant` should have relatively higher width, right?

Reply:
Yes, my bad!  it was 5.*rem Good check! @palashmon